velox/type/tests/VariantTest.cpp

```diff
@@ -111,14 +111,46 @@ TEST(VariantTest, shortDecimal) {
   // 0.1234 < 1.234
   EXPECT_LT(
       u2.value<TypeKind::SHORT_DECIMAL>(), v.value<TypeKind::SHORT_DECIMAL>());
+
+  // Hash test.
```

Comment:
This test is getting very long. Would you extract hash testing into a separate test?